run-automation-test.md

```diff
@@ -0,0 +1,145 @@
+# Running Automation test with Kobiton
+## Table of contents
++ [Prerequisites](#prerequisites)
++ [1. Configure TeamCity and integrate with GitHub](#1-configure-teamcity-and-integrate-with-github)
++ [2. Configure automation test script](#2-configure-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
++ [5. Feedback](#5-feedback)
+## Prerequisites 
+- Java(JRE). Supported are:
```

Comment:
Why do we even need Java ?

Reply:
Because TeamCity is based on Java, so it is necessary

Reply:
so Java need to install in teamCIty server or in our local ?  This doesn't show anythings

Reply:
In readme.md,  I said: "TeamCity is a Java-based build management and continuous integration server".  Do I need to explain more?

Reply:
Please read my question again!

Reply:
Please read my question again!

Reply:
yes, Java needs to run TeamCity sever